backend/models/Story.js

```diff
@@ -0,0 +1,85 @@
+module.exports = function(sequelize, Sequelize) {
+  return sequelize.define(
+    "stories",
+    {
+      id: {
+        type: Sequelize.INTEGER,
+        autoIncrement: true,
+        primaryKey: true
+      },
+      perspective: {
+        type: Sequelize.STRING
+      },
+      age: {
+        type: Sequelize.INTEGER
+      },
+      lgbtq: {
+        type: Sequelize.STRING
+      },
+      race: {
+        type: Sequelize.STRING
+      },
+      phone: {
+        type: Sequelize.STRING
+      },
+      topic: {
+        type: Sequelize.STRING
+      },
+      title: {
+        type: Sequelize.STRING
+      },
+      lede: {
```

Comment:
What is lede?

Reply:
keeping this, think of it like a headline